### ISSUE TITLE
Some parts of the pages are not visible to screen reader

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#362]
from #325 which has more details.

- [ ] investigate missing items from #325  - are other screen readers affected?

## Outcomes
ensure screen reader can now read out the following correctly:
### Page sections:
- [ ] Footer
- [ ] Green, gold and diamond open access boxes (who we are / open access movement pages)
- [ ] Award logos under grants and awards section on Who We Are.
### Filter:
- [ ] Resources page filter at the top
### Buttons:
- [ ] contact page - submit button
- [ ] login page - log in button
- [ ] register page - register button

**Comment by joemull** [comment]
Refinement: @StephDriver and @joemull are both going to test this with different assistive tech to work out what's going on here.

**Comment by joemull** [comment]
Screen reader pair programming:

- [ ] The open-access model infographic has `role="img"` which is preventing the normal sequence of TTS from catching the inner content: the h3s and ps. We need to remove this `role` and potentially also the `aria-label` on that same div. Then handle the structure differently for the a11y tree so that it is clear the OLH model is the Diamond model.
